diff --git a/bin/phantomas.js b/bin/phantomas.js
index 39b669a..f811825 100755
--- a/bin/phantomas.js
+++ b/bin/phantomas.js
@@ -9,6 +9,10 @@
  */
 'use strict';
 
+var EventEmitter = require('events').EventEmitter;
+var Log = require('elasticsearch/src/lib/log');
+Log.prototype.listenerCount = EventEmitter.prototype.listenerCount;
+
 var phantomas = require('..'),
 	async = require('async'),
 	debug = require('debug')('phantomas:cli'),
diff --git a/lib/engines/webkit2.js b/lib/engines/webkit2.js
index 6294929..de71c38 100644
--- a/lib/engines/webkit2.js
+++ b/lib/engines/webkit2.js
@@ -3,7 +3,15 @@
  *
  * @see https://github.com/macbre/phantomas/issues/488
  */
-var pkg = require('phantomjs-prebuilt');
+var pkg = {
+	name: 'PhantomJS',
+	engine: 'webkit2',
+	path: '/usr/local/bin/phantomjs',
+	version: '2.1.1',
+	getUserAgent: function() {
+		return 'PhantomJS/' + pkg.version
+	}
+};
 
 module.exports = {
 	name: 'PhantomJS',
diff --git a/package.json b/package.json
index 4ca66fa..41daafa 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,6 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "analyze-css": "^0.10",
     "ansicolors": "~0.3.2",
     "ansistyles": "~0.1.0",
     "ascii-table": "0.0.8",
@@ -36,11 +35,8 @@
     "node-statsd": "0.1.1",
     "node-uuid": "~1.4.1",
     "optimist": "0.6.x",
-    "phantomjs": "^1.9.15",
-    "phantomjs-prebuilt": "^2.1.3",
     "progress": "~1.1.4",
     "q": "^1.4.1",
-    "slimerjs": "^0.9.6",
     "tap": "^0.7.1",
     "tap-eater": ">=0.0.1",
     "travis-fold": ">=0.1.2"
